.github/workflows/deploy.yml

```diff
@@ -188,3 +146,47 @@ jobs:
           body_path: RELEASE.md
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  # Publish starship to Crates.io
+  cargo_publish:
```

Comment:
The jobs were reordered, with the only other change being made is that they all now have `needs: github_release`.